CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ endif()
 cmake_dependent_option(BUILD_WALLET_TOOL "Build bitcoin-wallet tool." ON "ENABLE_WALLET" OFF)
 
 option(THREADLOCAL "Enable features that depend on the C++ thread_local keyword (currently just thread names in debug logs)." ON)
-option(HARDENING "Attempt to harden the resulting executables." ON)
+option(ENABLE_HARDENING "Attempt to harden the resulting executables." ON)
```

Comment:
In commit 376f256307f4830e44668eebb177976e34bd9e09:

Is there a missing rename in the native_tidy ci job?